src/win/stream.c

```diff
@@ -48,6 +48,48 @@ int uv_listen(uv_stream_t* stream, int backlog, uv_connection_cb cb) {
 }
 
 
+int uv_reject(uv_stream_t* client) {
```

Comment:
Call this server, for consistency with Unix, and because we actually call it on the server.